### ISSUE TITLE
[TF2] [Feature Request] Disable Random Critical Hits in Medieval Mode.

### ISSUE THREAD

**Issue by AlxiumOfficial** [ValveSoftware/Source-1-Games#6208]
This feature would disable random critical hits in Medieval Mode.

Medieval mode is an alternative game mode in Team Fortress 2, where players are restricted to melee weapons along with a few other whitelisted weapons.

As with most casual matchmaking, random critical hits are enabled as well in Medieval mode. However, this raises several problems with game play that makes medieval mode matches feel less skill based and more on random chance. 

Melee weapons that can roll for random critical hits can feel frustrating and unfun to play against. Random critical hits on melee weapons cannot be detected by either player, due to melee weapons lacking hit-scan bullets or projectiles with which to apply a critical hit effect. Consequentially, there is no way for a player to know they are going to be hit with a random critical hit on melee weapons. This can be frustrating as it removes any chance for the player to respond to the random critical hit, with the only surefire way to avoid a death being not to engage in melee combat with the offensive player, which in itself is what Medieval Mode is all about.

Another effect of random critical hits in Medieval mode is that they often result in weapons that _can't_ roll for random critical hits being underutilized, such as the specialized Demoman swords. Some weapons like the Killing Gloves of Boxing and Scotsmann's Skullcutter are also more powerful than other options due to random critical hits; The Killing Gloves of Boxing can still roll for random critical hits, which on a successful kill rewards the player with guaranteed critical hits. The Skullcutter receives a 20% damage bonus and the movement speed penalty is negated as almost all Demoman players will have equipped boots and a shield instead of a grenade launcher and sticky bomb launcher. 

To make Medieval mode more fun for all and open up more opportunities to equip different weapons, random critical hits should be disabled by default in medieval mode.

**Comment by JoriKos** [comment]
> As with all casual matchmaking

Not Mannpower

**Comment by Tiagoquix** [comment]
Partial duplicate of #5291

**Comment by Flpstrike** [comment]
Random Criticals are the worst part of the entire game

**Comment by JoriKos** [comment]
> someone had to say this

Yeah I've never really heard anyone complain about this, very weird.

**Comment by DiskIntegrity** [comment]
Valve will never remove/nerf crits lmao, I feel like these requests are kinda pointless

**Comment by KimmyTF2** [comment]
> Valve will never remove/nerf crits lmao, I feel like these requests are kinda pointless

Random damage spread was also once a thing that really screwed up players, so who knows?

**Comment by gdfsgdfg** [comment]
No, random crits are part of tf2 always have been unless valve plans to do something with weapons that have the "no random criticals stat" which in case that would be a huge update.

**Comment by JoriKos** [comment]
> No, random crits are part of tf2 always have been

Just because it's in the game doesn't mean it's good.
